plugins/editorplaceholder/plugin.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @fileOverview Simple CKEditor 4 plugin that adds placeholder text to the editor.
+ */
+(function () {
+	CKEDITOR.plugins.add("editorplaceholder", {
+		isSupportedEnvironment: function () {
+			return !CKEDITOR.env.ie || CKEDITOR.env.version >= 9;
+		},
+
+		onLoad: function () {
+			CKEDITOR.addCss(CKEDITOR.plugins.editorplaceholder.styles);
+		},
+
+		init: function (editor) {
+			if (
+				!this.isSupportedEnvironment() ||
+				!editor.config.editorplaceholder
+			) {
+				return;
+			}
+
+			bindPlaceholderEvent(editor, "contentDom");
+			bindPlaceholderEvent(editor, "focus");
+			bindPlaceholderEvent(editor, "blur");
+
+			// Debounce placeholder when typing to improve performance (#5184).
+			bindPlaceholderEvent(
+				editor,
+				"change",
+				editor.config.editorplaceholder_delay
+			);
+		},
+	});
+
+	var ATTRIBUTE_NAME = "data-cke-editorplaceholder";
+
+	/**
+	 * Namespace providing the configuration for the Editor Placeholder plugin.
+	 *
+	 * @singleton
+	 * @class CKEDITOR.plugins.editorplaceholder
+	 * @since 4.15.0
+	 * @member CKEDITOR.plugins
+	 */
+	CKEDITOR.plugins.editorplaceholder = {
+		/**
+		 * Styles that would be applied to the editor by the placeholder text when visible.
+		 *
+		 * @property {String}
+		 */
+		styles:
+			"[" +
+			ATTRIBUTE_NAME +
+			"]::before {" +
+			"position: absolute;" +
+			"opacity: .8;" +
+			"color: #aaa;" +
```

Comment:
In HRDS, the placeholder color is different from the native one. So, can we provide a config to customise the placeholder colour?